.github/workflows/integration-tests.yml

```diff
@@ -609,3 +609,32 @@ jobs:
           ${SCCACHE_PATH} --show-stats
 
           ${SCCACHE_PATH} --show-stats | grep -e "Cache hits\s*[1-9]"
+
+  issues-1652:
+    runs-on: ubuntu-latest
+    needs: build
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [nightly-2023-02-08, nightly-2023-02-09]
```

Comment:
Is it necessary to hard-coded the date?

Reply:
I found it's broken since the version `nightly-2023-02-09` so I choose those two version to test.